osu.Game/Database/RealmContextFactory.cs

```diff
@@ -361,6 +361,17 @@ private void applyMigrationsForVersion(Migration migration, ulong targetVersion)
         private string? getRulesetShortNameFromLegacyID(long rulesetId) =>
             efContextFactory?.Get().RulesetInfo.FirstOrDefault(r => r.ID == rulesetId)?.ShortName;
 
+        public void CreateBackup(string filename)
+        {
+            using (BlockAllOperations())
+            {
+                Logger.Log($"Creating full realm database backup at {filename}", LoggingTarget.Database);
+                using (var source = storage.GetStream(Filename))
```

Comment:
kind of dangerous variable naming choices here, what with the `[fF]ilename` casing, but seems correct

Reply:
yeah, although it's a nit, I think it's safer to rename the parameter according to what it holds (i.e. `backupFilename`).

Reply:
I've renamed the variable.